aten/src/ATen/native/mkldnn/xpu/FusionUtils.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+#include <detail/oneDNN.h>
+
+namespace at {
+namespace native::xpu {
+xpu::onednn::Attr unary_attr_with_arg(
```

Comment:
Since it is under `xpu` namespace, `xpu::` should be redundant.

Reply:
has changed to `at::native::onednn`